scripts/cnode-helper-scripts/gLiveView.sh

```diff
@@ -151,6 +151,13 @@ if [[ ${UPDATE_CHECK} = Y && ${SKIP_UPDATE} != Y ]]; then
     1) $0 "$@" "-u"; myExit 0 ;; # re-launch script with same args skipping update check
     2) exit 1 ;;
   esac
+
+  # check for CNCLI update
+  if [ -z "${CNCLI_STATUS}" ] ; then CNCLI_STATUS=$(check_cncli_versions) ; fi
+  if [ -n "${CNCLI_STATUS}" ] ; then
+    printf "$CNCLI_STATUS\n\n" && read -p "Press key to ACK"
```

Comment:
We already have a function in env file loaded at startup to use for user interaction called `waitToProceed`.
The default message is good enough and in line with scripts in guild scripts library. So just call this command as is with no params and it will wait for keypress with text:
`press any key to proceed ..`

Reply:
Great, about to update this ASAP.